templates/courseware/progress.html

```diff
@@ -0,0 +1,238 @@
+<%inherit file="/main.html" />
+<%namespace name='static' file='/static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+from util.date_utils import get_time_display, DEFAULT_SHORT_DATE_FORMAT
+from django.conf import settings
+from django.utils.http import urlquote_plus
+%>
+<%block name="bodyclass">view-in-course view-progress</%block>
+
+<%block name="headextra">
+<%static:css group='style-course-vendor'/>
+<%static:css group='style-course'/>
+</%block>
+
+
+<%namespace name="progress_graph" file="/courseware/progress_graph.js"/>
+
+<%block name="pagetitle">${_("{course_number} Progress").format(course_number=course.display_number_with_default) | h}</%block>
+<%block name="nav_skip">#course-info-progress</%block>
+
+<%block name="js_extra">
+<script type="text/javascript" src="${static.url('js/vendor/flot/jquery.flot.js') | h}"></script>
+<script type="text/javascript" src="${static.url('js/vendor/flot/jquery.flot.stack.js') | h}"></script>
+<script type="text/javascript" src="${static.url('js/vendor/flot/jquery.flot.symbol.js') | h}"></script>
+<script type="text/javascript" src="${static.url('js/courseware/certificates_api.js') | h}"></script>
+<script type="text/javascript" src="${static.url('js/courseware/credit_progress.js') | h}"></script>
+<script>
+  ${progress_graph.body(grade_summary, course.grade_cutoffs, "grade-detail-graph", not course.no_grade, not course.no_grade) | h}
+</script>
+</%block>
+
+
+<%include file="/courseware/course_navigation.html" args="active_page='progress'" />
+
+<div class="container">
+  <div class="profile-wrapper">
+
+    <div class="course-info" id="course-info-progress" aria-label="${_('Course Progress')}">
+      % if staff_access and studio_url is not None:
+        <div class="wrap-instructor-info">
+          <a class="instructor-info-action studio-view" href="${studio_url | h}">${_("View Grading in studio")}</a>
+        </div>
+      % endif
+
+      <header class="progress-certificates">
+        <h1 class="progress-certificates-title">${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email) | h}</h1>
+      </header>
+
+      <section class="progress-info-mouseover">
+        <p>
+            ${_("""Only pre-test and post-test scores are tracked. You must
+                receive a score of 75% or higher on the post-test in order to eligibilie a certificate.
```

Comment:
Sure thing, we just copied verbatim the copy we got from mark, but we will take a look